core/tests/protractor_utils/PreferencesPage.js

```diff
@@ -70,6 +71,12 @@ var PreferencesPage = function() {
     correctOptions.first().click();
   };
 
+  this.setUserBio = function(bio) {
+    userBioElement.clear();
+    userBioElement.sendKeys(bio);
+    // pageHeader.click();
```

Comment:
Don't submit commented out code.

Reply:
Thank you for your comments! 
@nithusha21 i have commented this because of the problem I was telling to @DubeySandeep. 
Without clicking pageHeader(or any other space except UserBioElement), the text is not saved in userBioElement, and thus the tests do not pass. But when the script clicks it, the text is saved.
Can I use this pageHeader.click(), or can you suggest any other solutions?